src/lib/Server/Output/ValueObjectVisitor/Exception.php

```diff
@@ -115,12 +115,17 @@ public function visit(Visitor $visitor, Generator $generator, $data)
         $generator->startValueElement('errorMessage', $this->httpStatusCodes[$statusCode]);
         $generator->endValueElement('errorMessage');
 
-        if ($data instanceof Translatable && $this->translator) {
-            /** @Ignore */
-            $errorDescription = $this->translator->trans($data->getMessageTemplate(), $data->getParameters(), 'repository_exceptions');
+        if ($this->debug || $statusCode < 500) {
```

Comment:
should this not be interval comparison (400 < $code < 500) just to be sure or if this is handled somewhere higher?

Reply:
We want to use different messages only for >=500 errors. 4xx are "safe" errors i.e. unauthorized. Other status codes: 1xx, 2xx, 3xx aren't errors and are not visited by this visitor.